pytorch_lightning/metrics/functional/iou.py

```diff
@@ -21,7 +21,7 @@
 
 @deprecated_metrics(target=_iou)
 def iou(
-    pred: torch.Tensor,
+    preds: torch.Tensor,
```

Comment:
this will be handled with conditional wrapper https://github.com/Borda/pyDeprecate/pull/4